framework/auth/core.py

```diff
@@ -1122,6 +1122,10 @@ def get_daily_digest_log_ids(self):
         )
         return self.get_recent_log_ids(since=midnight)
 
+    def files_checked_out(self):
+        from website.files.models.osfstorage import FileNode
```

Comment:
This import needs to be called here to prevent an import error, due to circular import.